lib/active_merchant/billing/gateways/orbital.rb

```diff
@@ -886,12 +896,14 @@ def commit(order, message_type, retry_logic = nil, trace_number = nil)
             request.call(remote_url(:secondary))
           end
 
+        authorization = order.include?('<TokenTxnType>GT</TokenTxnType>') ? response[:safetech_token] : authorization_string(response[:tx_ref_num], response[:order_id])
```

Comment:
👗  Style:
This is fine but looks a bit complex due to the length of the line. Using a conventional if-else block might enhance readability but is up to you. 

```ruby
token_required = order.include?('<TokenTxnType>GT</TokenTxnType>')
authorization = if token_required
  response[:safetech_token]
else
  authorization_string(response[:tx_ref_num], response[:order_id])
end
```